SonarLint.VisualStudio.Integration.sln

```diff
@@ -4,6 +4,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 VisualStudioVersion = 17.0.31815.197
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Integration.Vsix", "src\Integration.Vsix\Integration.Vsix.csproj", "{FF2AD819-28F4-493A-8E9B-1D3F16BD4689}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
This tells MSBuild that the VSIX project depends on the new tool project.

It was set from the "Project Build Order" dialogue:

![image](https://user-images.githubusercontent.com/10757559/180183149-ba2a9e0c-940b-4174-9c19-f3f913720be6.png)